<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomObjectClass</class>
 <widget class="QDialog" name="CustomObjectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomObject</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设置男生年龄：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>设置女生年龄：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbBoyAge">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbGrilAge">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBoyInc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>男生长大一岁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGrilInc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>女生长大一岁</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMateInfo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>类的元对象信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearText">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空文本框</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CustomObject.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClearText</sender>
   <signal>clicked()</signal>
   <receiver>txtInfo</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
